modules/replication/dockerimage-replication/modulestack.yaml

```diff
@@ -2,18 +2,24 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: This stack deploys a Module specific IAM permissions
 
 Parameters:
-  # DeploymentName:
-  #   Type: String
-  #   Description: The name of the deployment
-  # ModuleName:
-  #   Type: String
-  #   Description: The name of the Module
   ProjectName:
     Type: String
     Description: The name of the project
   RoleName:
     Type: String
     Description: The name of the IAM Role
+  HelmDistroSecretName:
+    Type: String
+    Description: The name of the IAM Role
```

Comment:
nit: Descriptions wrong for `HelmDistroSecretName` and  `HostedRepoSecretName`

Reply:
on it